firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsUncaughtExceptionHandler.java

```diff
@@ -57,9 +57,7 @@ public void uncaughtException(Thread thread, Throwable ex) {
       Logger.getLogger().e("An error occurred in the uncaught exception handler", e);
     } finally {
       Logger.getLogger()
-          .d(
-              "Crashlytics completed exception processing."
-                  + " Invoking default exception handler.");
+          .d("Completed exception processing." + " Invoking default exception handler.");
```

Comment:
You can drop the "+" here, no?

Reply:
Yeah, this was a formatting thing. I'll fix it.